documentation/external_ssl_termination.markdown

```diff
@@ -5,39 +5,27 @@ canonical: "/puppetserver/latest/external_ssl_termination.html"
 ---
 
 
-In network configurations that require external SSL termination, there are some important differences between configuring the Apache/Passenger stack and configuring Puppet Server. Use the following steps to configure external SSL termination.
+Use the following steps to configure external SSL termination.
 
 ## Disable HTTPS for Puppet Server
 
 You'll need to turn off SSL and have Puppet Server use the HTTP protocol instead: remove the `ssl-port` and `ssl-host` settings from the `conf.d/webserver.conf` file and replace them with `port` and `host` settings. See [Configuring the Webserver Service](https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/blob/master/doc/jetty-config.md) for more information on configuring the web server service.
 
 ## Allow Client Cert Data From HTTP Headers
 
-When using external SSL termination, Puppet Server expects to receive client certificate information via some HTTP headers.
+When using external SSL termination, Puppet Server expects to receive client
+certificate information via some HTTP headers.
 
-By default, reading this data from headers is disabled.  To allow Puppet Server
-to recognize it, you'll need to set the `allow-header-cert-info` setting to `true`.
+By default, reading this data from headers is disabled.  To allow Puppet
+Server to recognize it, you'll need to set `allow-header-cert-info: true`
+in the `authorization` config section of the
+`/etc/puppetlabs/puppetserver/conf.d/auth.conf` file.
 
-Prior to the inclusion of `trapperkeeper-authorization` and an `auth.conf` file
-specific to Puppet Server, you would need to edit (or create) `conf.d/master.conf`
-and add `allow-header-cert-info: true` to the `master` config section.  See
-[Puppet Server Configuration](./configuration.markdown) for more information on
-the `master.conf` file.  This approach, however, is now deprecated.
+See [Puppet Server Configuration](./configuration.markdown) for more
+information on the `puppetserver.conf` and `auth.conf` files.
 
-Instead, it is preferred to enable `trapperkeeper-authorization` and
-set the `allow-header-cert-info` setting via the `authorization` config
-section.  This involves the following steps:
-
-* Migrate any custom authorization rule definitions that you may have made to core Puppet's
- `/etc/puppetlabs/puppet/auth.conf` file over to the
- `/etc/puppetlabs/puppetserver/conf.d/auth.conf` file.
-* Set the `jruby-puppet.use-legacy-auth-conf` setting in the
- `conf.d/puppetserver.conf` file to `false`.
-* Add `allow-header-cert-info: true` to the `authorization` config section in
- the `/etc/puppetlabs/puppetserver/conf.d/auth.conf` file.
-
-See [Puppet Server Configuration](./configuration.markdown) for more information
-on the `puppetserver.conf` and `auth.conf` files.
+Note: This assumes the default behavior of Puppet 5 and greater of using
+Puppet Server's hocon auth.conf rather Puppet's older ini-style auth.conf.
```

Comment:
* Add code tag for `auth.conf`. What does ini-style mean?

Reply:
"ini" is a format like "json", but unlike json there isn't a formal standard but instead its emerged from conventional use. And puppet doesn't follow the conventions either. Hence "ini-style".

It looks like this:
```
[section name]
  config_setting = config_value
  another_setting = another_value

# often how comments start
; sometimes how comments start
[another section]
  # The bits in the curly braces are puppet specific extensions
  setting = value { mode = 0644, owner = puppet, group = puppet }
```